packages/jest-core/src/collectHandles.ts

```diff
@@ -45,6 +47,24 @@ const hasWeakRef = typeof WeakRef === 'function';
 
 const asyncSleep = promisify(setTimeout);
 
+let gcFunc: Function | undefined = (globalThis as any).gc;
+function runGC() {
+  if (!gcFunc) {
+    v8.setFlagsFromString('--expose-gc');
+    gcFunc = vm.runInNewContext('gc');
+    v8.setFlagsFromString('--no-expose-gc');
+    if (!gcFunc) {
```

Comment:
Can this happen?

Reply:
I have not encountered. I'm not sure how to handle this situation in jest in general, warning or just throwing, what do you think?

Reply:
Throwing seems reasonable. It's essentially an invariant 👍